apps/prairielearn/src/models/user.ts

```diff
@@ -9,7 +9,7 @@ export async function selectUserById(user_id: string): Promise<User> {
   return await queryRow(sql.select_user_by_id, { user_id }, UserSchema);
 }
 
-export async function selectUserByUid(uid: string): Promise<User | null> {
+export async function selectOptionalUserByUid(uid: string): Promise<User | null> {
```

Comment:
Renamed to reflect our emerging convention that database functions that return `null` if something isn't found (as opposed to throwing an error) should be named `selectOptional...`.

Reply:
Love it! ❤️